Formula/picat.rb

```diff
@@ -0,0 +1,20 @@
+class Picat < Formula
+  desc "Simple, and yet powerful, logic-based multi-paradigm programming language"
+  homepage "http://picat-lang.org"
+  url "http://picat-lang.org/download/picat24_src.tar.gz"
+  version "2.4.8"
+  sha256 "72b452a8ba94d6187d837dcdb46aab0d7dc724651bac99a8cf2ada5c0a3543dd"
+  depends_on "gcc"
+
+  def install
+    inreplace "emu/Makefile.picat.mac64", "/usr/local/bin/gcc", HOMEBREW_PREFIX/"bin/gcc-8"
```

Comment:
The compiler should not be hardcoded in the makefile. Please report this to upstream.

Reply:
Yes, I'll write a mail to upstream. The build system seems not changed for a long time.